web/src/App.tsx

```diff
@@ -19,8 +19,8 @@ import "antd/dist/antd.css";
 function AppRoutes() {
   const routes = useRoutes([
     { path: "/", element: <RootPage /> },
-    { path: "/dashboard", element: <DashboardPage /> },
-    { path: "/dashboard/:workspaceId", element: <DashboardPage /> },
+    { path: "/dashboard", element: <WorkspacePage /> },
+    { path: "/dashboard/:workspaceId", element: <WorkspacePage /> },
```

Comment:
If it isn't a dashboard page any more please update the route as well

Reply:
@KaWaite , sure it is the main dashboard page, but it represents the workspace.
so for the route I think it is good to keep it this way.